src/main/kotlin/no/nav/syfo/behandlerdialog/MeldingFraBehandlerService.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.syfo.behandlerdialog
+
+import no.nav.syfo.behandlerdialog.domain.Melding
+import no.nav.syfo.behandlerdialog.kafka.KafkaMeldingFraBehandler
+import no.nav.syfo.personoppgave.*
+import no.nav.syfo.personoppgave.domain.PersonOppgaveType
+import java.sql.Connection
+
+fun processMeldingFraBehandler(
```

Comment:
Kunne denne ligget som en privat funksjon i KafkaMeldingFraBehandler-klassen? Den bruker jo loggeren fra den klassen og kalles bare derfra.

Reply:
Tilsvarende gjelder for `processUbesvartMelding` under

Reply:
Ja, jeg tenkte litt på det. Prøvde å følge et mønster som fantes i de andre consumerne, men akkurat her kan de like gjerne høre sammen 🤷🏼‍♂️